python/README.rst

```diff
@@ -21,10 +21,19 @@ Basic Usage
 
 
     sdk = looker_sdk.init31()  # or init40() for v4.0 API
-    # and away you go
     my_user = sdk.me()
-    new_user = looker_sdk.models.WriteUser(first_name="Jane", last_name="Doe")
-    sdk.create_user(body=new_user)
+
+    # output can be treated like a dictionary
+    print(my_user["first_name"])
+    # or a model instance (User in this case)
+    print(my_user.first_name)
+
+    # input methods can take either model instances like WriteUser
+    sdk.create_user(
+        body=looker_sdk.models.WriteUser(first_name="Jane", last_name="Doe")
+    )
+    # or plain dictionaries
+    sdk.create_user(body={"first_name": "Jane", last_name: "Doe"})
```

Comment:
Do you want to cover how extra properties are ignored?

Reply:
I want to keep it super brief for this basic usage. probably time to add some examples (both for dictionary and for enums)